conans/client/build/cppstd_flags.py

```diff
@@ -103,15 +103,19 @@ def _cppstd_visualstudio(visual_version, cppstd):
     v14 = None
     v17 = None
     v20 = None
+    v23 = None
 
     if Version(visual_version) >= "14":
         v14 = "c++14"
         v17 = "c++latest"
     if Version(visual_version) >= "15":
         v17 = "c++17"
         v20 = "c++latest"
+    if Version(visual_version) >= "17":
+        v20 = "c++20"
+        v23 = "c++latest"
 
-    flag = {"14": v14, "17": v17, "20": v20}.get(str(cppstd), None)
+    flag = {"14": v14, "17": v17, "20": v20, "23": v23}.get(str(cppstd), None)
```

Comment:
These changes are missing in ``conan.tools._compilers``, please make sure they are done there too.

Reply:
added